src/app/view/Lock/useLockScreen.ts

```diff
@@ -36,6 +37,17 @@ export const useLockScreenProps = (props: LockScreenProps): LockViewProps => {
   const { fqdn } = getInstanceAndFqdnFromClient(client)
   const { t } = useI18n()
 
+  // It is safer to start listening to backpress events in the UI layer.
```

Comment:
It is "safer" compared to what? Maybe this phrase is missing some context

Reply:
after thinking about it I think this comment doesn't add anything, removing it. the following sentence is useful though